static/js/public/manage-members.js

```diff
@@ -138,4 +140,77 @@ function updateMembers(membersData) {
   });
 }
 
-export { inviteMember, updateMembers };
+function filterMembers(members, roles) {
+  members = members || [];
+  roles = roles || [];
+
+  const ROLES = {
+    admin: "admin",
+    reviewer: "review",
+    viewer: "view",
+    publisher: "access",
+  };
+
+  const membersTableBody = document.querySelector("#members-table tbody");
+  const filterMembersField = document.querySelector("#filter-members");
```

Comment:
In order to avoid confusion and bugs, lately I've been using the `data` attribute to target elements. So in this case you could use:
```suggestion
  const filterMembersField = document.querySelector("[data-js='filter-members']");
```
In my view this will make you code more future proof and avoid issues with people changing class or id names. 
This is more a recommendation, rather than anything.

Reply:
In cases like these I think an ID is better as it indicates there will only be one of these elements. I do agree with using data attributes in place of classes though.